apps/docs/content/guides/auth/third-party/auth0.mdx

```diff
@@ -21,6 +21,15 @@ Auth0 can be used as a third-party authentication provider alongside Supabase Au
 
 ```typescript
 import { createClient } from '@supabase/supabase-js'
+import createAuth0Client from '@auth0/auth0-spa-js'
```

Comment:
i think the import is incorrect - shouldn't it be `Auth0Client`? (disclaimer: didn't really check the package if that method exists tho)

Reply:
ah good catch, from the repo description both methods exist but `createAuth0Client` is used as an async method - must have forgotten to change when I swapped out.

Thanks!